windows_containers/index.adoc

```diff
@@ -0,0 +1,33 @@
+[id="windows-container-overview"]
+= Windows container overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for {product-title} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. By using Red Hat subscription, you can get support for Windows Container. For more information see, xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[Release notes].
+
+xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Windows container workloads]:
+For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on Red Hat Enterprise Linux CoreOS (RHCOS) or Red Hat Enterprise Linux (RHEL).
+Windows container workloads support for clusters running on the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+
+xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Enable Windows container workloads]: You need a Machine Config Operator (WMCO) to run Windows workloads in your cluster. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
```

Comment:
s/You need a Machine Config Operator (WMCO)/You need the WMCO

You already spelled out the full name earlier.

Reply:
acked